ast/functions.go

```diff
@@ -330,7 +330,132 @@ type FuncCallExpr struct {
 
 // Restore implements Node interface.
 func (n *FuncCallExpr) Restore(ctx *RestoreCtx) error {
-	return errors.New("Not implemented")
+	switch n.FnName.O {
```

Comment:
Please `switch` using `n.FnName.L`, and add a test case showing that

```sql
extract(day from '1999-01-01')
```

restores correctly